pages/index.vue

```diff
@@ -1259,14 +1259,14 @@ export default {
         }
         return requestString.join("\n");
       } else if (this.requestType == "Fetch") {
-        var requestString = [];
-        var headers = [];
+        const requestString = [];
+        const headers = [];
```

Comment:
@jamesgeorge007 
```
Error: "headers" is read-only
```

Ran into this error on Generate Code modal

To reproduce:

0. Visit preview deployment: https://deploy-preview-343--postwoman.netlify.com
1. Make a change / send a request
2. Click "Generate Code"
3. Switch Request Type to `Fetch`
4. Other two (JavaScript and cURL) Request Types works fine. But Fetch code is not generated, and ran into error. Works on https://postwoman.io

Reply:
:+1: